modules/01 JavaScript Programming/Assignments/20/answer20.js

```diff
@@ -0,0 +1,16 @@
+
+
+function filter(arr){
+
+    return arr.filter(item=> item >6);
```

Comment:
The question's purpose is to write a custom "filter" function. So you shouldn't use the original "filter" in your solution